lumicks/pylake/kymotracker/kymoline.py

```diff
@@ -422,13 +422,23 @@ def save(self, filename, delimiter=";", sampling_width=None):
         """
         export_kymolinegroup_to_csv(filename, self._src, delimiter, sampling_width)
 
-    def fit_binding_times(self, n_components, *, exclude_ambiguous_dwells=True):
+    def fit_binding_times(
+        self, n_components, *, exclude_ambiguous_dwells=True, tol=None, max_iter=None
```

Comment:
`tol` and `max_iter` are not actually forwarded.

Reply:
Maybe add a test for this as well?

Reply:
good catch